DigitalLearningSolutions.Data/DataServices/NotificationDataService.cs

```diff
@@ -105,5 +107,24 @@ public IEnumerable<int> GetRoleBasedNotifications(int isCentreManager, int isCon
                 ).AsEnumerable();
         }
 
+        public void SubscribeDefaultNotifications(int? candidateId, int? adminUserId)
+        {
+            if (candidateId != null)
+            {
+                connection.Execute(
+                @"INSERT INTO NotificationUsers (
+                    NotificationID,CandidateID)

```

Comment:
We should join to the Notifications table here and honour the AutoOptIn flag (only insert a record if it is True) as per the existing logic for admin notification subscriptions.

Reply:
This is addressed.